configs/ot2_defconfig

```diff
@@ -8,6 +8,7 @@ BR2_TARGET_GENERIC_HOSTNAME="opentrons"
 BR2_TARGET_GENERIC_ISSUE="Welcome to the Opentrons OT2"
 BR2_INIT_SYSTEMD=y
 # BR2_TARGET_GENERIC_REMOUNT_ROOTFS_RW is not set
+BR2_GENERATE_LOCALE="C.UTF-8 en_US.UTF-8"
```

Comment:
Don't _really_ know if we need to generate en_US given this machine is laboratory equipment that's mainly meant to talk to other equipment / clients. Could keep/remove if anyone felt strongly either way

Reply:
The only reason not to, to me, would be file size, and the builds here don't look too much bigger.